Add CR4 folder with letter template based on RD suggestion in GDoc

samumantha opened the pull request:
Please check that everything important is mentioned in the template before merging and suggesting to partners to use it :)

samumantha committed:
Add letter template based on RD suggestion in GDoc

bast commented:
I will discuss this tomorrow at meeting with NeIC management and get their feedback.

samumantha committed:
first draft for organization info

to attract new partners :)

samumantha committed:
add headers

samumantha commented:
I am working now a bit more on info for organizations :hammer:

samumantha committed:
update partner info

bast committed:
wrap long lines for easier review/changes

bast committed:
reformulated one bullet point

because it started as benefit and went immediately into what the benefit
would be for everyone else, so I re-ordered

bast committed:
updated letter template borrowing from draft to CSC

I think this gives a more concrete starting point for existing
partners. For new partners we might need to modify a bit
but more important now is that we get support from existing partners.

samumantha committed:
Add other ways to join the fun

bast reviewed (approved):
This is really great.

bast commented:
OK if I merge so that we can send this out? I really like it.

samumantha commented:
With the new section about "other ways to join the fun", is there still enough good reasons in the becoming a partner section to make it attractive?

samumantha commented:
OK to merge from my side (please update links in team meeting hackmd)

bast commented:
IMO yes: partner means more influence and also steering. But the system relies that there are individuals and organizations who want to also create and not only consume. I will merge and adjust links.
